pkg/costmodel/allocation.go

```diff
@@ -58,6 +58,13 @@ const (
 	queryFmtLBActiveMins             = `count(kubecost_load_balancer_cost) by (namespace, service_name, %s)[%s:%s]`
 )
 
+// Constants for Network Cost Subtype
+const (
+	networkZoneCost     = "NetworkZoneCost"
+	networkRegionCost   = "NetworkRegionCost"
```

Comment:
Going to check if we should call these "CrossZone" and "CrossRegion" or "XZone" and "XRegion" instead of just "Zone" and "Region".

Reply:
Just saw Webb's comment. Let's go with "NetworkCrossRegionCost" and "NetworkCrossZoneCost"

Reply:
Addressed by updating the variable name and regenerating the codec file